commcare_connect/opportunity/tables.py

```diff
@@ -58,15 +59,26 @@ def show_warning(record):
 
 class UserVisitReviewFilter(FilterSet):
     review_status = ChoiceFilter(choices=VisitReviewStatus.choices, empty_label="All Reviews")
+    user = ModelChoiceFilter(queryset=User.objects.none(), empty_label="All Users", to_field_name="username")
```

Comment:
Im pretty confused by this user filter queryset behavior. Can you explain a bit more what is occurring? It is set to empty, then overwritten on `init`, but where do the values in the queryset for `id__in` come from, and why does it need to be done in such a way?

Reply:
The `ModelChoiceFilter` filter requires a query set of objects to be instantiated with, its a filter wrapper for `ModelChoiceField`, so I have used a `.none()` query set object to instantiate that filter as none query set don't make any queries to database. Further the `self.queryset` used in the `id__in` is the query set for UserVisits that this `UserVisitReviewFilter` works on and is passed into the filter when it is instantiated in the `user_visit_review` view. I hope this sounds good and please suggest any improvements you have to improve this.

Reply:
Thanks, it feels like there should be a better way, but not one I can think of.